plugins/inputs/procstat/procstat_test.go

```diff
@@ -369,3 +370,39 @@ func TestGather_cgroupPIDs(t *testing.T) {
 	assert.Equal(t, []PID{1234, 5678}, pids)
 	assert.Equal(t, td, tags["cgroup"])
 }
+
+func TestExitStatus(t *testing.T) {
+	out, err := exec.Command("pgrep", "-G", "sys").Output()
+	t.Logf("cmd out: %v", out)
+	exitNum, _ := internal.ExitStatus(err)
+	t.Logf("error status: %v", exitNum)
+	require.Equal(t, 1, exitNum)
+}
+
+func TestNoPgrepResults(t *testing.T) {
+	p := Procstat{
+		PidFinder: "pgrep",
```

Comment:
Same comment as above for this and the next test. You should be able to use testPgrep in this same file to fake the results.